secure-releases.md

```diff
@@ -0,0 +1,60 @@
+
+### Publishing
+If you have only 1 person who can publish releases, then this step is optional. If you have 2 or more, this step is strongly recommended.
+
+Publish a packed tarball from CI, only upon an explicit request. This means, don't make a new release automatically, but make sure that a human has intentionally said "it's time to publish a new version, here's the version number it will use". This explicit request can be a pushed semver-versioned git tag, or an explicit user-run workflow - but does not include merging/committing to the default branch.
+
+Do not publish without using 2FA. Note that an npm automation token is a single factor, and does not qualify as 2FA. Since Github does not provide any functionality to allow you to input a 2FA code natively, you'll need to use something like [Wombat Dressing Room](https://opensource.googleblog.com/2020/01/wombat-dressing-room-npm-publication_10.html) which requires you to host a service on GCP ([example](https://github.com/mathiasbynens/String.prototype.endsWith/blob/main/.github/workflows/publish-on-tag.yml#L21-L22)), or [Step Security's Wait For Secrets](https://github.com/step-security/wait-for-secrets) ([example](https://github.com/jsx-eslint/eslint-plugin-react/blob/28f9a6cdf716bc34f824ea3a3988e68f3e54eb0f/.github/workflows/npm-publish.yml)) which requires you to trust Step Security's external service.
+
+## CVE Management
+
+### Security Policy
+
+Your software project will, likely inevitably, have a security vulnerability. When (not if) that happens, you want to make sure that whoever discovers it has a way to report it to you privately - this is called "responsible disclosure" or "coordinated disclosure". So, the first important step is to *have a security policy*.
+
+On Github, this is always located at `$REPO_URL/security/policy` ([example](https://github.com/nvm-sh/nvm/security/policy)). This page can be populated from a number of file locations: `SECURITY.md` or `.github/SECURITY.md` in the repo itself, or, one of the same files in your user or organization’s `.github` repository. If you're not on Github, then look for a similar platform-specific convention, and if you find none, a top-level `SECURITY.md` is safe.
+
+In this policy, you'll want to mention a private reporting channel. This could be an email address, a service like HackerOne, etc.
+
+#### Programmatic Private Vulnerability Reporting
+
+Github and Gitlab support a Private Vulnerability Reporting (PVR) feature and API. This allows reporters to use the platform itself, programmatically if needed. It is strongly recommended to enable this feature, if it's not already on by default.
```

Comment:
perhaps mention this can be enabled at the org level for all repos, as well as a setting to automatically enable it for any new repos in the org